### ISSUE TITLE
Expose the --project option

### ISSUE THREAD

**Issue by rijk** [i18next/i18next-gettext-converter#57]
Can we expose this option in the command line parameters, so we can tweak the project name in the PO files?

https://github.com/i18next/i18next-gettext-converter/blob/master/src/lib/json2gettext.js#L62

**Comment by perrin4869** [comment]
Sure thing! Could you send a PR? Thanks!

**Comment by perrin4869** [comment]
I'll do this this week, stay tuned!

**Comment by perrin4869** [comment]
released in 4.0.0 :D

**Comment by rijk** [comment]
So weird.. When I use the `--project` parameter, it doesn't write the .po files anymore.. It says 'file written', but no path and the file is not written:

Normal:
```
rijk$ i18next-conv -l en -s ./i18n/en/translation.json -t ./i18n/en/translation.po
start converting
--> reading file from: ./i18n/en/translation.json
<-- writing file to: ./i18n/en/translation.po
file written
```

With new `--project` flag:
```
rijk$ i18next-conv -l en -s ./i18n/en/translation.json -t ./i18n/en/translation.po --project MyProj
start converting
--> reading file from: ./i18n/en/translation.json
file written
```

**Comment by perrin4869** [comment]
will take a look at this during the weekend!

**Comment by perrin4869** [comment]
Fixed in 4.0.2!

**Comment by rijk** [comment]
Sweet, confirm it's working now.
